dss/config.py

```diff
@@ -92,34 +92,40 @@ def set_config(config: BucketConfig):
         Config._CURRENT_CONFIG = config
 
     @staticmethod
-    def get_cloud_specific_handles(replica: str) -> typing.Tuple[
-            BlobStore, HCABlobStore, str]:
+    def get_cloud_specific_handles(replica: Replica) -> typing.Tuple[BlobStore, HCABlobStore, str]:
+
+        if type(replica) == str:
+            return Config.get_cloud_specific_handles(Replica[str(replica)])
 
         handle: BlobStore
-        if replica == 'aws':
+        if replica == Replica.aws:
             handle = S3BlobStore()
             return (
                 handle,
                 S3HCABlobStore(handle),
                 Config.get_s3_bucket()
             )
-        elif replica == 'gcp':
+        elif replica == Replica.gcp:
             credentials = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
             handle = GSBlobStore(credentials)
             return (
                 handle,
                 GSHCABlobStore(handle),
                 Config.get_gs_bucket()
             )
-        raise ValueError("I don't understand this replica!")
+        raise NotImplementedError(f"Replica `{replica.name}` is not implemented!")
 
     @staticmethod
-    def get_storage_schema(replica: str) -> str:
-        if replica == 'aws':
+    def get_storage_schema(replica: Replica) -> str:
+
+        if type(replica) == str:
+            return Config.get_storage_schema(Replica[str(replica)])  # noqa
```

Comment:
Why the cast `str(replica)`?

Reply:
@xbrianh otherwise mypy complains. It wasn't picking up the `# noqa` comment. I'll try again.

Reply:
I'd consider doing `instanceof` because you don't require exact type matching.  Generally, `instanceof` is preferable to inspecting the actual type.

Reply:
The `mypy` check was complaining and not picking up the `noqa` tag. I am going to try to remove this statement. It shouldn't be there in the first place.